### PR TITLE
Pass initial state onto Submissions Form

### LINKED ISSUE #24: Pass initial state onto submission form
If the project is deemed eligible, and the user clicks to make a submission, the answers to the nine questions that have been answered on the eligibility form should be transferred to the submission as the initial state (see [this](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/pages/form.js#L98), [this](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/pages/form.js#L105) and [this](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/pages/form.js#L227)) of that form and notify the user accordingly (see [this](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/pages/form.js#L106) and [this](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/pages/form.js#L199)), similar to this other notification (see blue box at the top):

Screen Shot 2021-06-18 at 12 28 24 PM

FYI: @nathanbaleeta

### PR DESCRIPTION
**Pull request by rachita19082**
Fix #24 
Pass the answers collected in the Eligibility form to the Submissions form.

### CODE REVIEW

File: packages/eligibility/src/Eligibility.js [hunk 1]
```diff
@@ -19,15 +21,56 @@ function Eligibility() {
...
+  };
+
+  const debouncedSave = useCallback(
+    debounce((vals) => saveToDb(vals), 1000),
+    [cookies.uuid]
+  );
+
+  async function saveToDb(vals) {
+    if (cookies.uuid) {
+      console.log("Enters saveToDb");
```
Comment (lacabra) [inline packages/eligibility/src/Eligibility.js:60]: Please remove `console.log()` statements from production code.

**Review by lacabra — CHANGES_REQUESTED** [review]
@rachita19082: Can you comment on any integration tests that you have done to validate that it works as expected? As far as I can see the code looks fine.

Also can you please explain the reasoning behind `fieldName` and which fields are you matching with the chosen nomenclature? 

Thanks! 🙏

**Comment by rachita19082** [comment]
> @rachita19082: Can you comment on any integration tests that you have done to validate that it works as expected? As far as I can see the code looks fine.
> 
@lacabra I have not performed any tests so far. One way to test the changes in the local environment would be to enable CORS while passing data from localhost. If [localhost:8080/eligibility/](https://localhost:8080/eligibility/) can also be included in this [pull request](https://github.com/lacabra/submission-digitalpublicgoods/pull/41) then I will be able to test if the code works as expected. Please let me know if there is some other way to test.

**Comment by rachita19082** [comment]
> Also can you please explain the reasoning behind `fieldName` and which fields are you matching with the chosen nomenclature?
> 
@lacabra I am assigning the chosen `Yes` or `No` values to the field names given in the [schema](https://github.com/lacabra/submission-digitalpublicgoods/blob/e8b231fa23c23e45722f6252c18f59e5bc6a76f5/schemas/schema.js) of the Submission Form so that the corresponding fields are initiated with the passed values.

File: packages/eligibility/src/Eligibility.js (hunk 1 above)
Comment (rachita19082) [reply packages/eligibility/src/Eligibility.js]: Resolved in commit 0edde98.

**Review by lacabra — APPROVED** [review]

**Comment by lacabra** [comment]
> @lacabra I have not performed any tests so far. One way to test the changes in the local environment would be to enable CORS while passing data from localhost. If [localhost:8080/eligibility/](https://localhost:8080/eligibility/) can also be included in this [pull request](https://github.com/lacabra/submission-digitalpublicgoods/pull/41) then I will be able to test if the code works as expected. Please let me know if there is some other way to test.

You can override the name resolution in your local machine, and hardcode the resolution of digitalpublicgoods.net to the loopback address 127.0.0.1 so that both pages are served from your local dev environment and appear as the same domain, which should remove the CORS issue

**Comment by rachita19082** [comment]
> You can override the name resolution in your local machine, and hardcode the resolution of digitalpublicgoods.net to the loopback address 127.0.0.1 so that both pages are served from your local dev environment and appear as the same domain, which should remove the CORS issue

I will try this fix. Thank you.

**Review by nathanfletcher — APPROVED** [review]

**Comment by nathanfletcher** [comment]
I am going ahead to merge changes.
Hopefully, the fix from [#41](https://github.com/lacabra/submission-digitalpublicgoods/pull/41) may fix it without the need for the loopback address fix mentioned [here](https://github.com/unicef/publicgoods-scripts/pull/34#issuecomment-883250143). If not, I will look forward to it.

**Comment by nathanfletcher** [comment]
> @rachita19082 I am going ahead to merge changes.
> Hopefully, the fix from [#41](https://github.com/lacabra/submission-digitalpublicgoods/pull/41) may fix it without the need for the loopback address fix mentioned [here](https://github.com/unicef/publicgoods-scripts/pull/34#issuecomment-883250143). If not, I will look forward to it.

This did not work as I missed a few things. So I created a [new PR](https://github.com/lacabra/submission-digitalpublicgoods/pull/45) to attempt another fix

**Pull request merged**